test/cmd_line/vsplit.test.ts

```diff
@@ -33,4 +33,25 @@ suite('Vertical split', () => {
 
     assertEqual(vscode.window.visibleTextEditors.length, 2, 'Editor did not split in 1 sec');
   });
+
+  test('Run :vsplit', async () => {
+    await commandLine.Run('vsplit', modeHandler.vimState);
+    await WaitForEditorsToClose(2);
+
+    assertEqual(vscode.window.visibleTextEditors.length, 2, 'Editor did not split in 1 sec');
+  });
+
+  test('Run :vnew', async () => {
+    await commandLine.Run('vnew', modeHandler.vimState);
+    await WaitForEditorsToClose(2);
+
+    assertEqual(vscode.window.visibleTextEditors.length, 2, 'Editor did not split in 1 sec');
+  });
+
+  test('Run :vne', async () => {
+    await commandLine.Run('vne', modeHandler.vimState);
+    await WaitForEditorsToClose(2);
```

Comment:
I'm looking at the implementation for `WaitForEditorsToClose`. Who is making it time out after 1s?

Reply:
Yes, it can be fixed with something like:

```typescript
export async function WaitForEditorsToClose(numExpectedEditors: number = 0, timeout: number = NaN): Promise<void> {
  let waitForTextEditorsToClose = new Promise((c, e) => {
    if (vscode.window.visibleTextEditors.length === numExpectedEditors) {
      return c();
    }

    vscode.window.onDidChangeVisibleTextEditors(() => {
      if (vscode.window.visibleTextEditors.length === numExpectedEditors) {
        c();
      }
    });

    if (!isNan(timeout)) {
      setTimeout(e, timeout);
    }
});
```

The `NaN` check should make it backwards compatible.